CONTRIBUTING.md

```diff
@@ -1,14 +1,22 @@
 # How to contribute
 
-Operator SDK is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, contact points for developers, and other resources to help get contributions into operator-sdk.
+Java Operator SDK is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, contact points for developers, and other resources to help get contributions into java-operator-sdk.
```

Comment:
```suggestion
Java Operator is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, contact points for developers, and other resources to help get contributions into the Java Operator.
```

Reply:
thank you for your comments. I modified according to above suggestion.